src/components/HomePage/MovieAndMovieList/Movie.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+import { MovieInfo } from '../../../types';
+import { AppText } from '../../shared/Text/AppText';
+import './Movie.css';
+import { BurgerMenu } from '../BurgerMenu/BurgerMenu';
+
+type MovieProps = {
+  movieInfo: MovieInfo
+};
+
+export const Movie: FC<MovieProps> = function (props) {
+  const { movieInfo } = props;
+  const {
+    posterPath, title, genre, year,
+  } = movieInfo;
+  const movieClass = classNames({
+    'd-flex': true,
```

Comment:
if it’s always true we do not need to set it.

something like this is enough:
const movieClass = classNames( 'd-flex','flex-row','justify-content-between','movie-details')

Reply:
ok, thanks, fixed